pyterrier/terrier/java.py

```diff
@@ -23,17 +23,20 @@
 
 @pt.java.before_init
 def set_version(version: Optional[str] = None):
-    configure['terrier_version'] = version
```

Comment:
I don't understand this logic -- what does this simplify? There's also no way to reset the terrier_version to default (latest version) anymore?

Reply:
I think we should have a different sentinal value for "use latest version" then.

Reply:
None is the latest version, right?

Reply:
yeah